test/t_two.ml

```diff
@@ -0,0 +1,33 @@
+open Lib.Two
+
+(** Framework utiltities *)
+let pp_codom ppf = function
+  | (x:int) -> Fmt.pf ppf "%d" x
+
+let eq_codom x y = (x=y)
+
+let codom_testable = Alcotest.testable pp_codom eq_codom
+
+(** Test utiltities *)
+let get_inputs =
```

Comment:
Suggest factoring out moves to enumerations to reduce input domain cardinality in exec function to input parsing function:

```
(** Test utiltities *)
[11:29 AM] Paasch, J. T.
type p1_move = A | B | C
type p2_move = X | Y | Z

let parse_tuple (move1, move2) = 
  let result_1 = match move1 with
    | 'A' -> A
    | 'B' -> B
    | 'C' -> C
  in
  let result_2 = match move2 with
    | 'X' -> X
    | 'Y' -> Y
    | 'Z' -> Z
  in
  (result_1, result_2)

let get_inputs =
  let filelines = String.split_on_char '\n' Two_input.site_input in
  let char_pairs = List.map ( fun line -> (line.[0], line.[2]) ) filelines
  List.map (fun tuple -> parse_tuple (tuple)) char_pairs
```